x-pack/plugins/ml/public/jobs/jobs_list/components/node_available_warning/node_available_warning.js

```diff
@@ -23,16 +24,39 @@ export function NodeAvailableWarning() {
     return (
       <React.Fragment>
         <EuiCallOut
-          title="No ML nodes available"
+          title={(<FormattedMessage
+            id="xpack.ml.jobsList.nodeAvailableWarning.noMLNodesAvailableTitle"
+            defaultMessage="No ML nodes available"
+          />)}
           color="warning"
           iconType="alert"
         >
           <p>
-            There are no ML nodes available.<br />
-            You will not be able to create or run jobs.
-            {isCloud &&
+            <FormattedMessage
```

Comment:
Let's make it as one label:
```js
         <FormattedMessage
              id="xpack.ml.jobsList.nodeAvailableWarning.noMLNodesAvailableDescription"
              defaultMessage="There are no ML nodes available.{nextLine}You will not be able to create or run jobs.{cloudConfigLink}"
              values={{
                 nextLine: <br/>,
                 cloudConfigLink: isCloud
                       ? <FormattedMessage
                  id="xpack.ml.jobsList.nodeAvailableWarning.linkToCloudDescription"
                  defaultMessage="This can be configured in Cloud {hereCloudLink}."
                  values={{
                    hereCloudLink: (
                      <EuiLink href="#">
                        <FormattedMessage
                          id="xpack.ml.jobsList.nodeAvailableWarning.linkToCloud.hereLinkText"
                          defaultMessage="here"
                        />
                      </EuiLink>
                    )
                  }}
                />
                       : '',
               }}
            />
```